rules/windows/process_creation/proc_creation_win_susp_appx_execution.yml

```diff
@@ -1,12 +1,13 @@
-title: Suspicious Windows App Activity
+title: Potential Suspicious Windows App Activity
 id: f91ed517-a6ba-471d-9910-b3b4a398c0f3
 status: experimental
-description: Detects suspicious children of application launched from inside the WindowsApps directory. This could be a sign of a rogue ".appx" package installation/execution
+description: Detects potential suspicious child process of applications launched from inside the WindowsApps directory. This could be a sign of a rogue ".appx" package installation/execution
```

Comment:
```suggestion
description: Detects potentially suspicious child process of applications launched from inside the WindowsApps directory. This could be a sign of a rogue ".appx" package installation/execution
```